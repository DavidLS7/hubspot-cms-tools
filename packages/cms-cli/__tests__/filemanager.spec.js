/*global banjo */

describe('Filemanager command', () => {
  it('works with help positional', async () =>
    expect(await banjo(['filemanager', 'help'])).toMatchSnapshot());

  it('works with help option', async () =>
    expect(await banjo(['filemanager', '--help'])).toMatchSnapshot());
});
