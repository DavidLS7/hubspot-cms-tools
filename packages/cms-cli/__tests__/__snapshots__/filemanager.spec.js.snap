// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filemanager command works with help option 1`] = `
"banjo filemanager

Commands for working with the File Manager

Commands:
  banjo filemanager upload <src> <dest>  Upload a folder or file from your
                                         computer to the HubSpot File Manager
  banjo filemanager fetch <src> [dest]   Download a folder or file from the
                                         HubSpot File Manager to your computer

Options:
  --version        Show version number                                 [boolean]
  --debug, -d      set log level to debug             [boolean] [default: false]
  --help           Show help                                           [boolean]
  --overwrite, -o  overwrite existing files           [boolean] [default: false]
  --config, -c     path to a config file                                [string]
  --portal, -p     HubSpot portal id or name from config                [string]
"
`;

exports[`Filemanager command works with help positional 1`] = `
"banjo filemanager

Commands for working with the File Manager

Commands:
  banjo filemanager upload <src> <dest>  Upload a folder or file from your
                                         computer to the HubSpot File Manager
  banjo filemanager fetch <src> [dest]   Download a folder or file from the
                                         HubSpot File Manager to your computer

Options:
  --version        Show version number                                 [boolean]
  --debug, -d      set log level to debug             [boolean] [default: false]
  --help           Show help                                           [boolean]
  --overwrite, -o  overwrite existing files           [boolean] [default: false]
  --config, -c     path to a config file                                [string]
  --portal, -p     HubSpot portal id or name from config                [string]
"
`;
